### PR TITLE
CCCP-3802: Update the code to consider luminous and nautilus

### PR DESCRIPTION
**Pull request by miloudbagaa**

### CODE REVIEW

File: files/ceph-version-qemu.sh [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
 # DEFAULTS
-# Search for upgrades from Jewel to Luminous.
-CEPH_MAJORVER_EXPECTED_OLD=10
-CEPH_MAJORVER_EXPECTED_NEW=12
+# Search for upgrades from Luminous to Nautilus.
+CEPH_MAJORVER_EXPECTED_OLD=12
+CEPH_MAJORVER_EXPECTED_NEW=14
 # QEMUs with no Rados Block Devices (RBD) are included in the results.
-EXCLUDE_QEMU_WITHOUT_RBD=1
+INCLUDE_QEMU_WITHOUT_RBD=1
```
Comment (Aegil) [inline files/ceph-version-qemu.sh]: Why the change

Comment (miloudbagaa) [reply files/ceph-version-qemu.sh]: The script by default considers all VMs that have or do not RBD. The VMs that do not have RBD are excluded only if the flag e is set (line 32). It is more logical to name the variable INCLUDE_QEMU_WITHOUT_RBD rather than EXCLUDE_QEMU_WITHOUT_RBD

Comment (Aegil) [reply files/ceph-version-qemu.sh]: Include / Exclude are opposite meanings. Changing the name of the variable implies it now does the opposite. There are no other changes, and no description to show why it should be this way. I don't think this sufficiently improves the code in a complete way.

Comment (miloudbagaa) [reply files/ceph-version-qemu.sh]: We can keep the name as it is but it does the opposite in the code. In the current version, EXCLUDE_QEMU_WITHOUT_RBD=1, this means that consider also the VMs without RBD.

Comment (miloudbagaa) [reply files/ceph-version-qemu.sh]: I have reverted the changes as previously.

Comment (Aegil) [reply files/ceph-version-qemu.sh]: Okay, looking through the code more completely I see the reasoning for this.

    e)
       EXCLUDE_QEMU_WITHOUT_RBD=0
       ;;
       
This was a missing change in the PR

**Review by Aegil — APPROVED** [review]
